.github/workflows/kfp-kubernetes-native-migration-tests.yaml

```
@@ -0,0 +1,195 @@
+name: KFP Kubernetes Native Migration Tests
+
+on:
+  push:
+    branches: [master]
```

That way, when we merge to RHDS, the CI runs on PRs and merges. Could you please ensure this is consistent on other workflows too?
```suggestion
    branches:
      - master
      - main
      - stable
      - 'rhoai-*'
```

---

Updated